### ISSUE TITLE
Notes on Beta Two Launch

### ISSUE THREAD

**Issue by wgranger** [zooniverse/anti-slavery-manuscripts#210]
- Field guide images take FOREVER to load
 - Cribsheet images only show a preview of the image object-fit: contain’ed into the box, making it tricky to see the entire image. Maybe the image should be clickable with a full-box view?
 - Tabbing isn’t set on all browsers (this needs to be explicitly stated in Firefox, etc?)
 - Toggle previous annotations should instead be a collaborative mode toggle

**Comment by beckyrother** [comment]
Agreed with crib sheet – maybe some modal on modal action a la twitter images?

**Comment by snblickhan** [comment]
Should refreshing the page bring up a new subject? Currently is not.

**Comment by beckyrother** [comment]
**transcribe interface:**
- [ ] It looks like the `m` when the transcribe box has been fixed, but `return` to close the box doesn't work anymore either. 

**elsewhere:**
**- about:**
- [ ] in the **nav**, the underline on hover effects the padding on the rest of the page, subtly but distractingly moving the content of the page down a few pixels
- [ ] should we include zooniverse bios
**- field guide:**
- [ ] align everything with the title of the modal:
![anti-slavery_manuscripts](https://user-images.githubusercontent.com/1392130/34732123-e6ee1e36-f529-11e7-9ee6-861ff144bab6.png)

**Comment by snblickhan** [comment]
@beckyrother Zooniverse bios are in the next merge, I thinK? there's an issue open for Team Bios right now.

**Comment by beckyrother** [comment]
About page: Make Charles Lenox Remond image larger:
![bpl-about](https://user-images.githubusercontent.com/1392130/34784366-a7970d9a-f5f3-11e7-8265-210825587285.png)

**Comment by snblickhan** [comment]
We need a way to navigate back to the Classify screen from Recents & Collections -- those tabs don't work to take users back to antislaverymanuscripts.org; they try to send users to the PB URL, which doesn't exist.

**Comment by eatyourgreens** [comment]
Have you set a redirect to `antislaverymanuscripts.org` on the project admin page? `project.redirect` is used for the classify tab, if it's set.

**Comment by beckyrother** [comment]
On the home page, the caption under the bell image should say `Detail from envelope addressed to the Westons, ca. 1832-1867`
